test/views/behaviour.js

```diff
@@ -0,0 +1,193 @@
+
+/**
+ * Behaviour of the Echidna testbed.
+ *
+ * @author Antonio <antonio@w3.org>
+ */
+
+'use strict';
```

Comment:
Why? `'use strict';` is code, and so is close to the code; the documentation comment at the beginning of a file documents the entire file (including that statement), not only the function/class following it.